src/System.Private.CoreLib/src/System/Reflection/Emit/ILGenerator.cs

```diff
@@ -38,23 +36,17 @@ internal static T[] EnlargeArray<T>(T[] incoming, int requiredSize)
             return temp;
         }
 
-        private static byte[] EnlargeArray(byte[] incoming)
+        private void EnlargeArray(int requiredSize)
```

Comment:
This is asymmetric with other EnlargeArray methods in this file now. Since it is called from a single place only, it does not need to be a separate method anymore. It can be inlined into the one place that it is called from.

Reply:
https://github.com/dotnet/coreclr/pull/23833/commits/e55ce5573cc5e634cb91b0379cfa7e0b37f90063#diff-9251654972afbdb424e02431cb2b1e3cR270